apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -84,84 +68,66 @@ class MultipleChoiceOverviewTable extends Component {
     });
   };
 
-  getColumns = (sortable) => {
-    let dataColumns = [
-      {
-        property: 'question',
-        header: {
-          label: commonMsg.question(),
-          props: {
-            style: {
-            ...tableLayoutStyles.headerCell,
-          }},
-          transforms: [sortable],
-        },
-        cell: {
-          props: {
-            style: {
-            ...tableLayoutStyles.cell,
-          }}
-        }
+  answerOptions = (index) => (
```

Comment:
naming here too: something like `getAnswerColumn` or `createAnswerColumn` could imply you're actually making one.

Reply:
Fixed.